src/assets/locales/en/translations.json

```diff
@@ -123,9 +123,10 @@
   "services.platform.list.infrastructure": "Infrastructure & Automation",
   "services.consulting.title": "Consulting",
   "services.consulting.teaser": "The experience of our employees is our most valuable treasure.",
-  "services.consulting.text": "The experience of our employees is our most valuable treasure. We share it with you and advise you in technical and conceptual matters. conceptual matters.",
```

Comment:
oh a classic deepl error I think. it tends to repeat things at the end of sentences